### PR TITLE
Fix padding in holistic rubrics

### PR DESCRIPTION
**Pull request by MitchelPaulin**

### CODE REVIEW

**Review by devpow112 — APPROVED** [review]

File: src/components/editor/rubric-criteria-editor.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ class RubricCriteriaEditor extends composeMixins(
 				}
 
 				.drag-handle-spacer {
-					width: calc(var(--d2l-rubric-editor-start-gutter-width) + 0.5rem);
+					margin-right: 2.6rem;
```
Comment (devpow112) [inline src/components/editor/rubric-criteria-editor.js:135]: Actually does this need an RTL version?

Comment (MitchelPaulin) [reply src/components/editor/rubric-criteria-editor.js:135]: Probably, whats the easiest way to force the rubric to display in RTL?

Comment (devpow112) [reply src/components/editor/rubric-criteria-editor.js:135]: The demo's have a toggle for it. In the LMS it's probably easiest to switch to arabic.

Comment (devpow112) [reply src/components/editor/rubric-criteria-editor.js:135]: Looks like there is also a way to enable pseudo RTL
https://d2l.slack.com/archives/C166RTA2H/p1595962377391900

Comment (MitchelPaulin) [reply src/components/editor/rubric-criteria-editor.js:135]: Kind of found a hack, added `document.documentElement.setAttribute('dir', 'rtl');` in one of the top level components constructors. Actually it seems to work fine :) 

image

**Review by devpow112 — APPROVED** [review]

**Pull request merged**